lib/modules/manager/pip_requirements/artifacts.ts

```diff
@@ -25,10 +28,13 @@ export async function updateArtifacts({
       .split(newlineRegex)
       .map((line) => line.trim());
     for (const dep of updatedDeps) {
+      const depName = dep.depName!;
```

Comment:
```suggestion
      if(!dep.Name) {continue;}
      const depName = dep.depName;
```

Reply:
Fine with me. The existing code base used the non-null assertion operator, that's why I kept it.

Reply:
Well, almost. Line 31 must read `if(!dep.depName) {continue;}`. :smile:

Reply:
resolved in d540d0cf1fdb3e2621920edc50790f38ad65150e